packages/webapp/pages/onboarding.tsx

```diff
@@ -146,10 +147,12 @@ export function OnboardPage(): ReactElement {
 
     return router.replace({
       pathname: '/',
-      query: {
-        welcome: 'true',
-        hset: 'true',
-      },
+      query: onboardingOptimizations
+        ? {}
+        : {
+            welcome: 'true',
+            hset: 'true',
+          },
```

Comment:
Not important, but different suggestion:

```suggestion
     ...(onboardingOptimizations && {
    query: {
      welcome: 'true',
      hset: 'true',
    },
  }
```

Reply:
We'll most propably add something to the query params here for share tracking which means we would need to change this back to the previous. But yeah until then, can change to this.

Reply:
Ah ok yeah then it's fine either way